libindy/src/errors/mod.rs

```diff
@@ -360,3 +426,17 @@ impl<E> IndyErrorExt for E where E: Fail
     }
 }
 
+thread_local! {
+    pub static LAST_ERROR: RefCell<Option<CString>> = RefCell::new(None);
```

Comment:
Does it need to be public?

Reply:
we can get access to thread variable only inside `with` closure.
we can avoid making LAST_ERROR public by adding setter which accepts `*mut *const c_char` param (in my opinion this worse).